### PR TITLE
Fix 4411

### PR DESCRIPTION
**Pull request by ismailsunni**
### What does it fix?
* Ticket: #4411 
* Funded by: DFAT
* Description: 
  - Fix the resizing's effect in the field mapping step
expanding minimum wizard

cc @samnawi @Charlotte-Morgan 

### Checklist:
- [x] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [x] Request someone to review or test your PR

### CODE REVIEW

File: safe/gui/tools/wizard/wizard_dialog.py [hunk 1]
```diff
@@ -876,3 +880,29 @@ def save_current_keywords(self):
...
+        else:
+            vertical_size_policy = QSizePolicy.Preferred
+
+        size_policy = QSizePolicy()
+        size_policy.setHorizontalPolicy(QSizePolicy.Preferred)
+        size_policy.setVerticalPolicy(vertical_size_policy)
+        self.scrollAreaWidgetContents.setSizePolicy(size_policy)
+        # Also reset the max height. 190 is a number that make it pretty
+        max_height = self.height() - 190
+        self.scrollAreaWidgetContents.setMaximumHeight(max_height)
```
Comment (ismailsunni) [inline safe/gui/tools/wizard/wizard_dialog.py]: __[flake8]__
* [W292] no newline at end of file

**Comment by Gustry** [comment]
Also fixing this ticket: https://github.com/inasafe/inasafe/issues/3533 Two tickets for one PR

File: safe/gui/tools/wizard/wizard_dialog.py [hunk 2]
```diff
@@ -876,3 +880,29 @@ def save_current_keywords(self):
...
+        :param event: The re-sized event.
+        :type event: QResizeEvent
+        """
+        self.resized.emit()
+        return super(WizardDialog, self).resizeEvent(event)
+
+    def after_resize(self):
+        """Method after resizing the window."""
+        if self.get_current_step() == self.step_kw_fields_mapping:
+            # Use MinimumExpanding when on the Field Mapping Step
```
Comment (Gustry) [inline safe/gui/tools/wizard/wizard_dialog.py]: Hum, why only this page? Can't it be extended to all other pages? cf the ticket I put

Comment (ismailsunni) [reply safe/gui/tools/wizard/wizard_dialog.py]: Ah yes. Thanks. I have made it applied to all step. Turns out, the most important one is set the max height.

**Comment by Gustry** [comment]
Mea culpa, it's finally 3 tickets for one PR: https://github.com/inasafe/inasafe/issues/2975

**Pull request merged**